homeassistant/components/edl21/sensor.py

```diff
@@ -39,13 +41,17 @@
 
 DOMAIN = "edl21"
 CONF_SERIAL_PORT = "serial_port"
+CONF_SCAN_INTERVAL_SECONDS = "scan_interval_seconds"
 MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=60)
 SIGNAL_EDL21_TELEGRAM = "edl21_telegram"
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
         vol.Required(CONF_SERIAL_PORT): cv.string,
         vol.Optional(CONF_NAME, default=""): cv.string,
+        vol.Optional(
+            CONF_SCAN_INTERVAL_SECONDS, default=MIN_TIME_BETWEEN_UPDATES
+        ): cv.positive_int,
```

Comment:
Scan intervals for platforms are already configurable out of the box.

See: https://www.home-assistant.io/docs/configuration/platform_options#scan-interval

Additionally, we no longer allow integrations to add or change a platform YAML configuration. Nor do we allow configuring a scan interval anymore.

See ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>

Reply:
Thank you, i did not find that info on my own